pkgs/by-name/se/seiri/package.nix

```
@@ -0,0 +1,31 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  nix-update-script,
+}:
+
+rustPlatform.buildRustPackage (finalAttrs: {
+  pname = "seiri";
+  version = "0.2.1";
+
+  src = fetchFromGitHub {
+    owner = "tarolling";
+    repo = "seiri";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-bAf8btrNnWCq4eTlssa+o456rWkBfhXaypsai1OT1Ws=";
+  };
+
+  cargoHash = "sha256-qsyrYxdGtjiJmV1KuCLq8MpdZ+KxMIzLe4HKxNxcXK8=";
+
+  passthru.updateScript = nix-update-script { };
+
+  meta = {
+    description = "Language-agnostic project visualization tool";
+    homepage = "https://github.com/tarolling/seiri";
+    changelog = "https://github.com/tarolling/seiri/releases/tag/${finalAttrs.src.tag}";
```

https://github.com/NixOS/nixpkgs/issues/367739
```suggestion
    changelog = "https://github.com/tarolling/seiri/releases/tag/v${finalAttrs.version}";
```

---

Is this required style by Nixpkgs right now?  I understand the override issue, but I much prefer `src.tag` over `version` to be honest.

---

It's not required, the discussion about which one to use treewide is still on going. Either would be fine in the meantime but I prefer `finalAttrs.version`.